### PR TITLE
Absolute value in short vectors (iteration)

### PR DESCRIPTION
**Pull request by StevellM**

### CODE REVIEW

File: test/QuadForm/ShortVectors.jl [hunk 1]
```diff
@@ -45,7 +45,7 @@
                      2448, 4320, 2982, 2402, 2364, 512, 256, 306 ]
 
   # Floating point bounds
-  L = integer_lattice(gram = QQ[1//2 0; 0 1//3])
```
Comment (thofma) [inline test/QuadForm/ShortVectors.jl:48]: I would duplicate the tests instead of removing the old tests.

**Comment by thofma** [comment]
The "absolute quadratic form" sounds a bit odd, as does the "absolute squared length". But I guess it is what it is. You have an opinion on the terminology @simonbrandhorst?

File: test/QuadForm/ShortVectors.jl (hunk 1 above)
Comment (StevellM) [reply test/QuadForm/ShortVectors.jl:48]: Good point, I will modify this.

File: src/QuadForm/ShortVectors.jl [hunk 2]
```diff
@@ -136,7 +151,8 @@ end
 @doc raw"""
     kissing_number(L::ZZLat) -> Int
 
-Return the Kissing number of the sphere packing defined by `L`.
+Return the Kissing number of the sphere packing defined by the absolute value
+of the quadratic form defined by `L`.
```
Comment (simonbrandhorst) [inline src/QuadForm/ShortVectors.jl:155]:
```suggestion
Return the Kissing number of the sphere packing defined by `L`.
```

**Review by simonbrandhorst — APPROVED** [review]

**Pull request merged**